go/pkg/api/handlers.go

```diff
@@ -3,12 +3,19 @@ package api
 import (
 	"encoding/json"
 	"net/http"
+
+	"github.com/shapeshift/unchained/internal/log"
 )
 
+var logger = log.WithoutFields()
+
 func HandleResponse(w http.ResponseWriter, status int, res interface{}) {
 	w.Header().Set("Content-Type", "application/json")
 	w.WriteHeader(status)
-	json.NewEncoder(w).Encode(res)
+	err := json.NewEncoder(w).Encode(res)
```

Comment:
prefer 1 liner err check where applicable. I am also wondering if we should bother with writing a generic error reponse/status code if we hit this error case. We have static response types, so I am not sure if it would be worth it. Do you have any opinion on additional handling here based on your experience so far?

Reply:
these errors couldn't be handled downstream so I don't think it's necessary to create static error types for these.